spec/registry.md

```diff
@@ -90,6 +80,11 @@ the `:string` selector function performs as described below.
 
 The `:string` function returns the string value of the _resolved value_ of the _operand_.
 
+> [!NOTE]
+> The function `:string` does not perform Unicode Normalization of its formatted output.
```

Comment:
No, since case folding would be a surprising operation for `:string` to perform.

Reply:
Should we add "... or its _resolved value_"?

Reply:
The resolved value isn't necessarily a string, though (if the non-normative example in the spec is followed, it would be an object with methods, one of which produces the formatted output).